deploy/charts/harvester/templates/configmap.yaml

```diff
@@ -145,3 +145,76 @@ data:
       sleep 2
     done
     `}}
+  cpu-manager.sh: |-
+    echo "Start update cpu-manager-policy option..."
+    KUBECTL="$HOST_DIR/$(readlink $HOST_DIR/var/lib/rancher/rke2/bin)/kubectl"
+    CPU_MANAGER_CONFIG_FILE="$HOST_DIR/etc/rancher/rke2/config.yaml.d/99-z01-harvester-cpu-manager.yaml"
+    STATIC_POLICY="static"
+    NONE_POLICY="none"
+    CPU_MANAGER_STATE_FILE="$HOST_DIR/var/lib/kubelet/cpu_manager_state"
+    NODE_NAME="$1"
+    NODE_POLICY="$2"
+    EXIT_CODE=0
+
+    if [ "$NODE_POLICY" != "$STATIC_POLICY" ] && [ "$NODE_POLICY" != "$NONE_POLICY" ]; then
+      echo "Error: invalid cpu-manager-policy $NODE_POLICY"
+      exit 1
+    fi
+
+    if ! $KUBECTL get node "$NODE_NAME" --show-labels | grep -q "cpumanager="; then
+      echo "Error: There is no label cpumanager in node $NODE_NAME."
+      exit 1
+    fi
+
+    printf 'kubelet-arg+:\n- "cpu-manager-policy=%s"' "$NODE_POLICY" > $CPU_MANAGER_CONFIG_FILE
+
+    if [ -f "$CPU_MANAGER_STATE_FILE" ]; then
+      mv "$CPU_MANAGER_STATE_FILE" "${CPU_MANAGER_STATE_FILE}.old"
+      echo "File $CPU_MANAGER_STATE_FILE has been renamed to ${CPU_MANAGER_STATE_FILE}.old"
+    else
+      echo "File $CPU_MANAGER_STATE_FILE does not exist."
+    fi
```

Comment:
Discussed offline with @ibrokethecloud , the flow of restart kubelet should be
- stop kubelet
- rm cpu_manager_state file
- start kubelet

Reply:
> we should in future iterations may be improve the helper script to bring the steps more in line with the documentation https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/#changing-the-cpu-manager-policy

Already addressed in the latest commit, also rebased the feature branch.